lib/bech32/test/Codec/Binary/Bech32Spec.hs

```diff
@@ -273,18 +273,33 @@ validChecksums = map B8.pack
     , "split1checkupstagehandshakeupstreamerranterredcaperred2y9e3w"
     ]
 
-invalidChecksums :: [ByteString]
-invalidChecksums = map B8.pack
-    [ " 1nwldj5"
-    , "\DEL1axkwrx"
-    , "an84characterslonghumanreadablepartthatcontain\
-      \sthenumber1andtheexcludedcharactersbio1569pvx"
-    , "pzry9x0s0muk"
-    , "1pzry9x0s0muk"
-    , "x1b4n0q5v"
-    , "li1dgmt3"
-    , "de1lg7wt\xFF"
-    ]
+invalidChecksums :: [(ByteString, Bech32.DecodingError)]
+invalidChecksums =
+            [ ( " 1nwldj5"
```

Comment:
Sorry to be annoying with this, but let's make this 4-space indent :grimacing: ... Have you configured your editor to pick up the configuration in `.editorconfig` at the root of the project?

Reply:
done. (I must have hit _tab_ too many times...)